test/Visualization_Tests/src/Id_Spec.enso

```diff
@@ -21,7 +21,7 @@ spec = Test.group "Serializable Visualization Identifiers" <|
         expected p_name v_name =
             lib = Json.from_pairs [["name", p_name]]
             Json.from_pairs [["library", lib], ["name", v_name]]
-        v_1 = Visualization.Id.from_module here "My Vis"
+        v_1 = Visualization.Id.from_module self "My Vis"
```

Comment:
Not sure on this but if it passes one to revisit later.

Reply:
```suggestion
        v_1 = Visualization.Id.from_module Id_Spec "My Vis"
```

Cannot we refer to the current module just be its name like so?

Reply:
Thanks @radeusgd, that works